addon/components/tf-form/component.js

```diff
@@ -0,0 +1,30 @@
+/**
+  @module ember-ticketfly-fields
+ */
+import Ember from 'ember';
+import layout from './template';
+const { Component, guidFor, computed } = Ember;
+
+/**
+  @public
+  @class TfForm
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+  
+  tagName: 'form',
+  classNames: ['tf-form'],
+
+  inputGuid: computed({
+    get() {
+      return `id-${guidFor(this)}`;
+    }
+  }),
+
+  describedByGuid: computed({
+    get() {
+      return `described-by-${guidFor(this)}`;
+    }
+  })
+});
```

Comment:
@BrianSipple Going with `tf-field` instead of `tf-form-field` since this doesn't necessarily need to be used in a form.